doc/release/1.17.0-notes.rst

```diff
@@ -163,6 +163,11 @@ thereby saving a level of indentation
 In some cases where ``np.interp`` would previously return ``np.nan``, it now
 returns an appropriate infinity.
 
+Pathlib support for `np.fromfile`, `ndarray.tofile` and `ndarray.dump`
+----------------------------------------------------------------------
+`np.fromfile`, `np.ndarray.tofile` and `np.ndarray.dump` now support
+the `pathlib.Path` type for the `file`/`fid` parameter.
+
```

Comment:
Restructured text is confusing.
The note does not quite [render](https://6236-908607-gh.circle-artifacts.com/0/home/circleci/repo/doc/build/html/release.html#pathlib-support-for-np-fromfile-ndarray-tofile-and-ndarray-dump) properly. If you want the objects to be links, they should use the numpy namespace `numpy.fromfile`, `numpy.ndarray.tofile`, `numpy.ndarray.dump`. If they should render as code, then they should have double back-ticks and the text can be anything. `file` and `fid` should have double back-ticks.

Reply:
Fixed :)